sklearn/preprocessing/imputation.py

```diff
@@ -189,6 +194,13 @@ def _sparse_fit(self, X, strategy, missing_values, axis):
         else:
             X = X.tocsc()
 
+        if axis == 0:
+            mask = _get_mask(X.data, self.missing_values)
+            mask_matrix = X.__class__((mask, X.indices.copy(),
+                                       X.indptr.copy()), shape=X.shape,
+                                      dtype=X.dtype)
+            self.feat_with_missing_ = mask_matrix.sum(axis=0).A.nonzero()[1]
```

Comment:
Yeah sorry my bad. I will send it as a return value, but the reason I stopped from doing that was currently the return value ``statistic`` is sent at the end of evaluation of the code path below and also ``feat_with_missing_`` is only makes sense for ``axis=0``.